src/MessageHandler/ZipDatasetFilesHandler.php

```diff
@@ -91,27 +91,27 @@ public function __construct(
      */
     public function __invoke(ZipDatasetFiles $zipDatasetFiles)
     {
-        $this->logger->info('Zipping files started');
         $fileIds = $zipDatasetFiles->getFileIds();
         $datasetSubmissionId = $zipDatasetFiles->getDatasetSubmissionId();
         $datasetSubmission = $this->entityManager->getRepository(DatasetSubmission::class)->find($datasetSubmissionId);
         $filesInfo = $this->fileRepository->getFilePathNameAndPhysicalPath($fileIds);
         $destinationPath = $this->downloadDirectory . DIRECTORY_SEPARATOR .  str_replace(':', '.', $datasetSubmission->getDataset()->getUdi()) . '.zip';
+        $this->logger->info('Zipping started: ' . $destinationPath);
         try {
             $outputStream = array('fileStream' => fopen($destinationPath, 'w+'));
             $this->zipFiles->start($outputStream, basename($destinationPath));
             foreach ($filesInfo as $fileItemInfo) {
+                $this->logger->info("adding file to $destinationPath:" . $fileItemInfo['filePathName']);
                 $this->zipFiles->addFile($fileItemInfo['filePathName'], $this->datastore->getFile($fileItemInfo['physicalFilePath']));
             }
             $this->zipFiles->finish();
+            $this->logger->info('Zipping ended: ' . $destinationPath);
             fclose($outputStream['fileStream']);
             $datasetSubmission->getFileset()->setZipFilePath($destinationPath);
             $this->entityManager->flush();
         } catch (\Exception $exception) {
             $this->logger->error(sprintf('Unable to zip file. Message: %s', $exception->getMessage()));
             return;
         }
-
-        $this->logger->info(sprintf('Zipping files done'));
```

Comment:
Similar to comment above. You can say that the worker is done.

Reply:
It seems that me a Praneeth were saying the same thing. But I feel the start worker messages should be the first line of code. What if things break before them.